File: crates/forge_services/src/tool_services/syn/lang/c/invalid.c
#include <stdio.h>
#include <stdlib.h>

typedef struct {
    char* name;
    int age;
    double score;
} Student;

int main() {
    Student students[3];
    
    // Initialize students
    students[0] = (Student){"Alice", 20, 95.5};
    students[1] = (Student){"Bob", 21, 87.0};
    students[2] = (Student){"Charlie", 19, 92.3};
    
    printf("Student List:\n");
    for (int i = 0; i < 3; i++) {
        printf("%s: Age %d, Score %.1f\n", 
               students[i].name, students[i].age, students[i].score);
    }
    
    // Missing return statement
    missing_semicolon_here
    
    // Missing closing brace for main

File: crates/forge_services/src/tool_services/syn/lang/swift/invalid.swift
import Foundation

class Person {
    let name: String
    var age: Int
    
    init(name: String, age: Int) {
        self.name = name
        self.age = age
    }
    
    func celebrateBirthday() {
        age += 1
        print("Happy Birthday \(name)! You are now \(age) years old.")
    }
}

class Student: Person {
    var grades: [Double]
    
    init(name: String, age: Int, grades: [Double]) {
        self.grades = grades
        super.init(name: name, age: age)
    }
    
    func averageGrade() -> Double {
        return grades.reduce(0, +) / Double(grades.count)
    }
}

let student = Student(name: "Alice", age: 20, grades: [95.0, 87.5, 92.3])
student.celebrateBirthday()
print("Average grade: \(student.averageGrade()")

// Missing closing parenthesis